build.sh

```diff
@@ -20,6 +20,18 @@ check_for python
 check_for svn
 check_for unzip
 check_for xml2-config
+check_for bash
+check_for java
+check_for javac
+check_for xzdec
+
+case `uname` in
+    OpenBSD)
+        check_for egcc
+        check_for eg++
+	;;
+esac
```

Comment:
That's not as bad a situation as I thought actually. Regardless, the C benchmarks at the very least need to be built with the same GCC. And if we go that far, we may aswell use this GCC to build the VMs. Will raise a bug.